trieste/models/gpflow/models.py

```diff
@@ -1362,3 +1368,305 @@ def covariance_between_points(
             query_points_2=query_points_2,
             whiten=whiten,
         )
+
+
+class AR1(TrainableProbabilisticModel):
```

Comment:
since we haven't been using abbreviations in naming our models/wrappers, lets be consistent and use a full name here - this stands for autoregressive, right? so perhaps `AutoRegressive1`, or as a compromise, since it s a multi-fidelity model, perhaps we could have `MultiFidelityAR1`